README.md

```diff
@@ -100,7 +139,9 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is a blueprint of a database. It defines how data in a database is constructed.
+
```

Comment:
that's correct! How can we represent a `one-to-many` relationship in a schema? Foreign Keys.